index.html

```diff
@@ -151,1145 +151,1103 @@ <h4>Other Related Resources</h4>
+              <img style="width: 130px; height: 200px;" src="images/taliq.jpg" alt="Taʻlīq script">
 
-      <figure id="figure-order_when_displayed">
-        <img src="images/order-when-displayed.svg" alt="The order of characters when displayed">
+              <figcaption>
+                Taʻlīq example [<a href=
+                "https://upload.wikimedia.org/wikipedia/commons/thumb/8/87/Ta'liq_script_1.jpg/389px-Ta'liq_script_1.jpg">Source</a>].
+              </figcaption>
+            </figure>
 
-        <figcaption>
-          The order of characters when displayed
-        </figcaption>
-      </figure>
 
+            <p><em>Taʻlīq</em> (hanging) is a beautiful script characterized by the precision and
+            stretch of its letters, its clarity, and its lack of complexity. Designed for Persian
+            language, until replaced by <em>Nastaʻlīq</em>.</p>
+          </dd>
 
-      <p>An initial step of the process involves determining each paragraph’s <span class=
-      "qterm">base direction</span>: whether the paragraph is left-to-right or right-to-left. The
-      base direction is either explicitly set by the author, inherited from the page, or (typically
-      for user-generated content) detected based on the content of the paragraph. The base
-      direction has two important uses later in the process.</p>
 
+          <dt>Dīwānī (ديواني)</dt>
 
-      <p>The next step is to split the text into <span class="qterm">directional runs</span>. Each
-      directional run is a sequence of characters with the same direction.</p>
 
+          <dd class="flexContainer">
+            <figure class="floatedFigure">
+              <img style="width: 250px; height: 101px;" src="images/diwani.png" alt=
+              "Diwani script">
 
-      <figure id="figure-directional_runs">
-        <img src="images/directional-runs.svg" alt="Splitting a text into 3 directional runs">
+              <figcaption>
+                Dīwānī example [<a href=
+                "https://commons.wikimedia.org/wiki/File:Izzet_44.png">Source</a>].
+              </figcaption>
+            </figure>
 
-        <figcaption>
-          Splitting a text into 3 directional runs
-        </figcaption>
-      </figure>
 
+            <p>Used by the Ottoman court (<em>Diwan</em>) to write official documents. Some
+            variations of it are still in use today (e.g. hand written documents by some religious
+            officials).</p>
+          </dd>
+        </dl>
 
-      <p>Inside each run, all the characters follow the same order. The runs themselves are ordered
-      for visual representation from left to right or from right to left, depending on the base
-      direction of the paragraph. <a href="#figure-order_of_directional_runs"></a> demonstrates an
-      example of this. This is the first effect of the base direction.</p>
 
+        <p>We can add other font styles to this list, such as the following :</p>
```

Comment:
maybe remove this, so it becomes a simple list, since there’s no indication as to why there is a separation anyway.

alternatively, group the items in some way – especially if it's useful for people who want to understand what’s relevant for the Web today (such as for the recent discussion about fallback fonts, where some people pointed at this section)

Reply:
Yes, this is not my text but came up as new because of reorganisation.